autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -243,6 +262,20 @@ def fit(train_data,
                     Note: a list of hyper-parameters dicts can be passed; each set will create different version of the model.
                     Note: Hyperparameter tuning is disabled for this model.
                     Note: 'penalty' parameter can be used for regression to specify regularization method: 'L1' and 'L2' values are supported.
+                Advanced functionality: Custom AutoGluon model parameters
```

Comment:
Maybe don't call these 'parameters' to avoid confusion the model's ML parameters, and the hyperparameters.   Could call them 'settings' or 'arguments' instead.

Reply:
Reworded to arguments